src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CcdCaseAssignment.java

```diff
@@ -54,6 +68,22 @@ public CCDCallbackResponse applyNoc(
                 createHeaders(serviceAuthorizationToken, userToken)
             );
 
+        log.info("authToken: {}", userToken);
+
+        ObjectMapper mapper = new ObjectMapper()
```

Comment:
I can see the same mapper configuration in JacksonConfiguration, can we not reuse it?

Reply:
Removed as had it there to log out json for debugging. Wrongly committed